From c7e36abe266a725fdda1084dcffef5182cde4cbb Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 26 Jul 2013 12:36:05 +0800
Subject: [PATCH 11/12] ENGR00272743 Remove duplicated function as it defined
 in jb4.3

param_get_min/param_get_max already been defined in jb4.3

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 pcm.c | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/pcm.c b/pcm.c
index a79da1d..f5eafe5 100644
--- a/pcm.c
+++ b/pcm.c
@@ -133,24 +133,6 @@ static unsigned int param_get_int(struct snd_pcm_hw_params *p, int n)
     return 0;
 }
 
-static unsigned int param_get_min(struct snd_pcm_hw_params *p, int n)
-{
-    if (param_is_interval(n)) {
-        struct snd_interval *i = param_to_interval(p, n);
-            return i->min;
-    }
-    return 0;
-}
-
-static unsigned int param_get_max(struct snd_pcm_hw_params *p, int n)
-{
-    if (param_is_interval(n)) {
-        struct snd_interval *i = param_to_interval(p, n);
-            return i->max;
-    }
-    return 0;
-}
-
 static void param_set_rmask(struct snd_pcm_hw_params *p, int n)
 {
     p->rmask |= 1 << n;
-- 
1.8.0

